domains/kagari.is-not-a.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "GitHub Pages for is-not-a.dev",
+    "domain": "is-not-a.dev",
+    "subdomain": "kagari",
+
+    "owner": {
+        "repo": "https://github.com/kagarifuyukawa/kagarifuyukawa.github.io",
+        "email": "fuyukawabot@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "kagarifuyukawa.github.io",
```

Comment:
Cannot have CNAME records in conjunction with other records and vice versa.

Review guidelines for PRs.